corehq/messaging/scheduling/views.py

```diff
@@ -1070,24 +1072,28 @@ def post(self, request, *args, **kwargs):
                     deleted=False,
                 )
             except AutomaticUpdateRule.DoesNotExist:
-                errors.append(_("Row {index}, with rule id {id}").format(index=index, id=row['id']))
+                messages.error(request, _("Could not find rule for row {index}, with id {id}.").format(
```

Comment:
I'm not familiar with this `messages` object. Does this change mean that errors will be reported one by one rather than accumulated and all at once?

Reply:
Nope, the messages will accumulate and then be displayed together, each in its own alert box.